### ISSUE TITLE
Parallelization does not work in R 4.0 + Rstudio

### ISSUE THREAD

**Issue by askerdb** [Russel88/DAtest#13]
There is a work-around in this thread, maybe it should be the default for now?

https://github.com/rstudio/rstudio/issues/6692

**Comment by Russel88** [comment]
Thanks for reporting this Asker.

It seems that Henriks solution should work without breaking things:
```
if (Sys.getenv("RSTUDIO") == "1" && !nzchar(Sys.getenv("RSTUDIO_TERM")) && 
    Sys.info()["sysname"] == "Darwin" && getRversion() == "4.0.0") {
  parallel:::setDefaultClusterOptions(setup_strategy = "sequential")
}
```
Can you confirm, because then I will include it

**Comment by askerdb** [comment]
It seemingly worked for me, but I can't find documentation on what changes.

**Comment by Russel88** [comment]
All that is changed is that the workers are not set up in parallel, but sequentially, and only on macOS Darwin, R 4.0 in Rstudio.

Since it works, I'll include it

**Comment by Russel88** [comment]
Should be fixed in 2.7.16
